demos/dlgr/demos/bartlett1932/templates/consent.html

```diff
@@ -41,5 +41,6 @@
                 <button type="button" id="no-consent" class="btn btn-danger btn-lg">No thanks (exit HIT)</button>
             </div>
         </div>
+        <script src="/static/scripts/tracker.js"></script>
```

Comment:
(This is labeled as being for testing purposes. Ditto for the other imports of this script below.)

Reply:
Yes, now that we have an official working demo of the tracking feature, it probably makes sense to remove it from the Bartlett demo. Once this is merged along with @davisagli's new template support we can allow web tracking to be enabled automatically for experiments via a global config. We can maybe even include it with dallinger2.js in a single combined JS module to be loaded in those cases.